addons/l10n_es_edi_tbai/wizards/account_invoice_refund_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_move_reversal" model="ir.ui.view">
+        <field name="name">account.move.reversal.form</field>
```

Comment:
`...inherit.l10n_es_edi_tbai` + fix xml_id

Reply:
xml_id should not be the same as inherited model id ?

Reply:
xml_id should not be the same as inherited model id ?